src/lib.rs

```diff
@@ -5,6 +5,7 @@
 #![test_runner(testing::sdk_test_runner)]
 #![allow(incomplete_features)]
 #![feature(generic_const_exprs)]
+#![feature(const_cmp)]
```

Comment:
Did not find this nighlty feature in [Unstable Book](https://doc.rust-lang.org/beta/unstable-book/the-unstable-book.html).

1. How did you find it ?
2. Where is it used ?

Reply:
@yogh333 it's for `let min_length = core::cmp::min(bytes.len(), MAX_METADATA_STRING_SIZE - 1);`

Reply:
But with @yhql 's proposed solution for strings, we probably would not need it.

Reply:
And for how I found it, I can't remember ^^

Reply:
We can remove this if we replace this `min` by a more manual version:
```
   let min_length = if bytes.len() <= MAX_METADATA_STRING_SIZE - 1 {
        bytes.len()
    } else {
        MAX_METADATA_STRING_SIZE - 1
    };
```

Reply:
Removed in https://github.com/LedgerHQ/ledger-device-rust-sdk/pull/96